src/hooks/useGetDocumentTitlePrice.ts

```diff
@@ -0,0 +1,19 @@
+import { useEffect } from 'react'
+import useGetPriceData from './useGetPriceData'
+
+const useGetDocumentTitlePrice = () => {
+  const priceData = useGetPriceData()
+  const cakePriceUsdNumber = priceData ? Number(priceData.prices.Cake) : Number.NaN
+
+  const cakePriceUsdString = Number.isNaN(cakePriceUsdNumber)
+    ? ''
+    : ` - $${cakePriceUsdNumber.toLocaleString(undefined, {
+        minimumFractionDigits: 3,
+        maximumFractionDigits: 3,
+      })}`
+
+  useEffect(() => {
+    document.title = `PancakeSwap${cakePriceUsdString}`
+  })
```

Comment:
this effect will rerun on each render, causing some serious performances issues

Reply:
done